topics/introduction/slides/introduction.html

```diff
@@ -238,32 +312,47 @@
 - Create **sub**-workflows: a workflow inside another workflow
 - **Share** workflows for publication and with the community
 
+???
+
+Potential information overload for newbies
```

Comment:
Can you elaborate?

Reply:
Just wrote this in the speaker notes as something that you might want to skip when presenting, if everyone's eyes are glazing over because they've never seen galaxy before and they've seen a huge amount of information. Sure there are more advanced things like sub workflows, or re-running with changed parameters but it's not necessary to elaborate on this in intro slides.

Reply:
As the slide title implies, this is meant as giving reasons why a user should bother using a workflow instead of just running tools one after the other. No need to explain each point in technical detail, and can be skipped if the attendees are already excited about workflows, but can be quite useful if they are skeptical instead.